css/style.css

```diff
@@ -79,7 +81,8 @@ img {
 .container {
   width: 1200px;
   margin: 0 auto;
-  padding: 0 10px;
+  box-shadow: 0px 5px 15px rgba(var(--shadow), 0.2),
+  0px 0px 0px 40px var(--special-grey);
```

Comment:
- Нам тут ни к чему двойная тень, в макете тень одна:
![image](https://user-images.githubusercontent.com/30810655/107934185-cbcb3c80-6f90-11eb-9e51-d0938b5ef169.png)
```suggestion
  box-shadow: 0 5px 15px rgba(var(--shadow), 0.2);
```
Также обрати внимание, что фон всей страницы не белый:
![image](https://user-images.githubusercontent.com/30810655/107934981-cd493480-6f91-11eb-9f32-19538cd10760.png)


- обертка всей страницы у нас всего одна, чтобы не путать ее с другими возможными обертками на странице стоит назвать более уникально, например `.page`